### ISSUE TITLE
A problem when reading 

### ISSUE THREAD

**Issue by rangfeng1** [NVlabs/motion-policy-networks#26]
Sorry to bother you again, I have a question. In the paper, I saw that you did experiments on MPNet, but I found that their data set was not available. How did you implement it at that time? Do you have relevant data? Woolen cloth?

**Comment by fishbotics** [comment]
That's a good question! I re-implemented an analogous dataset to the original MPNets dataset (same number of environments, same number of problems, same network input). I didn't open source this dataset or the model, but if you are interested in using it, I can. I implemented the model by copying it from their repo and I implemented the data by reading the details in the paper. Unfortunately, the data they released is for a Baxter robot which makes it hard to compare against MPiNets (the Baxter is no longer sold or officially fixed and our Baxter at UW is long broken).

**Comment by rangfeng1** [comment]
Yes, I am more interested in this because I am a newbie just learning, if possible, can you share the data, model and related details of the work you have done? thank you very much!
